gobblin-metrics-libs/gobblin-metrics-base/src/main/java/org/apache/gobblin/metrics/event/EventSubmitter.java

```diff
@@ -54,6 +52,10 @@
     private final Map<String, String> metadata;
     private final String namespace;
 
+    public Builder(MetricContext metricContext) {
```

Comment:
I though all event submitters need a namespace?

Reply:
Removed this.